packages/access-token/package.json

```diff
@@ -14,11 +14,14 @@
     "test:xs": "exit 0",
     "lint": "run-s --continue-on-error lint:*",
     "lint-fix": "yarn lint:eslint --fix",
-    "lint:eslint": "eslint ."
+    "lint:eslint": "eslint .",
+    "lint:types": "tsc"
   },
   "dependencies": {
-    "@agoric/assert": "^0.6.0",
+    "@types/n-readlines": "^1.0.3",
+    "@types/proper-lockfile": "^4.1.2",
```

Comment:
Shouldn't these be `devDependencies` (sorry missed it the first time around) ?

Reply:
Ah, yes, they should be.